docs/sources/installation/cruxlinux.md

```diff
@@ -60,6 +60,13 @@ or use it as part of your `FROM` line in your `Dockerfile(s)`.
 There are also user contributed [CRUX based image(s)](https://registry.hub.docker.com/repos/crux/) on the Docker Hub.
 
 
+## Uninstall Docker
+
+To uninstall the Docker package:
+
+    # prt-get remove docker
```

Comment:
`#` prompt indicates that this is run as `root`. To be consistent, I think it should be `$ sudo ..`